xbmc/cores/VideoPlayer/DVDCodecs/DVDFactoryCodec.cpp

```diff
@@ -54,10 +55,10 @@ CDVDVideoCodec* CDVDFactoryCodec::CreateVideoCodec(CDVDStreamInfo &hint, CProces
     hint.externalInterfaces->GetAddonInstance(ADDON::IAddonProvider::INSTANCE_VIDEOCODEC, addonInfo, parentInstance);
     if (addonInfo && parentInstance)
     {
-      pCodec.reset(new CAddonVideoCodec(processInfo, addonInfo, parentInstance));
+      pCodec = std::make_unique<CAddonVideoCodec>(processInfo, addonInfo, parentInstance);
       if (pCodec && pCodec->Open(hint, options))
```

Comment:
Don't need to check pCodec here for bool, as `std::make_unique()` will throw if `new` fails.